regtests/ww3_tp2.22/input/ww3_shel.inp

```diff
@@ -0,0 +1,42 @@
+$ WAVEWATCH III shell input file
+$ ------------------------------
+   F T
+   T T
+   T T
+   F F
+   F F
+   F F
+   F
+   F
+   F
+$
+   19680606 000000
+   19680606 120000
+$
+   1
+$
+   19680606 000000  10800  19680608 000000
+$
+  N
+  HS EF  T01 DIR SPR
```

Comment:
Add Stoke's Drift and other relevant output to this test to properly show new features capability and test to ensure things are working properly and that we do not break this feature in the future.